source/educators/concepts/accessibility/accessibility_guidelines.rst

```diff
@@ -69,9 +71,9 @@ disabilities, learners whose native language is not English, and learners with
 technical issues such as low bandwidth internet or no access to audio.
 
 To supplement the
-accessibility of the materials you can create within our platform, we recommend that
-you engage the resources available at your institution to support learners
-with disabilities. Most institutions offer disability support services and
+accessibility of the materials you can create within the platform, it is recommended that
```

Comment:
just curious about this one. I'm assuming there's a conscious choice to move away from "we"  basically meaning the project is recommending?  But I'm twitchy about  explicitly moving towards passive voice. Could this just be a directive, instead? eg. 
```
To supplement the
accessibility of the materials you can create within the platform, engage with any available
resources at your institution to support learners with disabilities.
```

Reply:
There's no clear "we" in the docs anymore (previously "we" specifically referred to edx.org). I like your rewording.